runelite-client/src/main/java/net/runelite/client/plugins/notes/NotesConfig.java

```diff
@@ -31,6 +31,11 @@
 @ConfigGroup("notes")
 public interface NotesConfig extends Config
 {
+	String CONFIG_GROUP = "notes";
+	String NOTES = "notes";
+
+	int MAX_NOTES = 5;
```

Comment:
id prefer this to be a config option so you can set said amount

Reply:
I didn't make it configurable (or unlimited) because if you have more than one lines of tab it would break the UI, so I just make sure the user cannot overflow the tab bar in the first place

Reply:
Make the range 1-5 then, so you can choose the max tabs they can even open :)

Reply:
Do you mean user should be able to set how many tabs they would have?

If so, initially user will have 1 note tab. They can click the + icon to add more tabs (see screenshot above). The + icon will disappear once they have added all 5 tabs.

Reply:
The limit is hardest to 5 so if you hit 5, the + goes
If you use the above config option with a range of 1-5, and the user sets it to 3, the + will disappear at 3

Reply:
I've added configurable max note pages.